articles/validating-xml-using-xsd/index.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/validating-xml-using-xsd/
+title: Validating XML using XSD
+description: A tutorial on validating XML using XSD
+author: srishilesh-p-s
+date: 2020-11-15T00:00:00-12:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/validating-xml-using-xsd/hero.jpg
+    alt: Validating XML using XSD
+---
+
+In this article, we will understand how validation of XML is done using XSD. By the end of this article, you will get an overview of various techniques for validating XML tags and attributes using XSD. You will also be learning step by step guide for validation. It is highly recommended to go through my [previous article](), to understand the basics of XML and its importance of validation.
```

Comment:
**NOTE:** The previous article (#532) refers to the PR for which the review is under progress.